addons/mail/static/src/messaging.js

```diff
@@ -243,9 +254,10 @@ export class Messaging {
         if (id in this.partners) {
             return this.partners[id];
         }
-        const partner = { id, name };
+        const partner = { id, name, im_status: "" };
         this.partners[id] = partner;
-        return partner;
+        this.updateImStatusRegistration();
+        return reactive(partner);
```

Comment:
You should not return `reactive()` here. All `Messaging` state is a reactive already (see `MessagingService`)